src/main/resources/config/application.yml

```diff
@@ -39,10 +39,15 @@ hystrix:
 management:
     security:
         roles: ROLE_SYS_ADMIN
+        enabled: false
     context-path: /management
     health:
+        diskspace:
```

Comment:
could you add a comment that we're disabling this because it is not authenticated and poses a small security issue?